--- accessrules.wsdl	2022-07-21 17:51:42.000000000 +0800
+++ accessrules-2.wsdl	2022-07-21 18:05:06.000000000 +0800
@@ -37,7 +37,7 @@
 		<xs:schema targetNamespace="http://www.onvif.org/ver10/accessrules/wsdl"
 				   xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:pt="http://www.onvif.org/ver10/pacs"
 				   xmlns:tar="http://www.onvif.org/ver10/accessrules/wsdl" elementFormDefault="qualified" version="18.12">
-			<xs:import namespace="http://www.onvif.org/ver10/pacs" schemaLocation="../../pacs/types.xsd"/>
+			<xs:import namespace="http://www.onvif.org/ver10/pacs" schemaLocation="../../ver10/pacs/types.xsd"/>
 			<!--ServiceCapabilities definitions-->
 			<xs:complexType name="ServiceCapabilities">
 				<xs:annotation>
